api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml

```diff
@@ -31,46 +30,46 @@
 <concept concept_id="1074" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2005-01-06 00:00:00.0" version="" changed_by="1" date_changed="2016-04-07 04:18:42.0" retired_by="1" uuid="1074AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
 <concept concept_id="1098" retired="false" datatype_id="4" class_id="23" is_set="false" creator="1" date_created="2005-01-12 00:00:00.0" version="" changed_by="1" date_changed="2013-12-27 20:47:37.0" retired_by="1" uuid="1098AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
 <concept concept_id="1099" retired="false" datatype_id="4" class_id="23" is_set="false" creator="1" date_created="2005-01-12 00:00:00.0" version="" changed_by="1" date_changed="2013-12-27 20:47:56.0" retired_by="1" uuid="1099AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept concept_id="1498" retired="false"  datatype_id="4" class_id="5" is_set="false" creator="1" date_created="2007-08-29 13:26:20.0" version="" changed_by="1" date_changed="2015-11-05 01:05:41.0" retired_by="1" uuid="1498AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept concept_id="1499" retired="false"  datatype_id="4" class_id="5" is_set="false" creator="1" date_created="2007-08-29 13:27:07.0" version="" changed_by="1" date_changed="2015-11-05 01:06:04.0" retired_by="1" uuid="1499AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept concept_id="1500" retired="false"  datatype_id="4" class_id="5" is_set="false" creator="1" date_created="2007-08-29 13:27:25.0" version="" changed_by="1" date_changed="2015-11-05 01:06:39.0" retired_by="1" uuid="1500AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept concept_id="1513" retired="false"  datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2007-10-21 22:05:03.0" version="" changed_by="1" date_changed="2014-05-16 13:56:54.0" retired_by="1" uuid="1513AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept concept_id="1514" retired="false" datatype_id="4" class_id="11" is_set="false" creator="1" date_created="2007-10-21 22:08:35.0" version="" changed_by="1" date_changed="2014-05-15 21:01:47.0" retired_by="1" uuid="1514AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
-<concept concept_id="1518" retired="false" datatype_id="4" class_id="25" is_set="false" creator="1" date_created="2007-10-21 22:15:30.0" version="" changed_by="1" date_changed="2014-05-15 20:18:17.0" retired_by="1" uuid="1518AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
+<concept concept_id="1498" retired="false" form_text="" datatype_id="4" class_id="5" is_set="false" creator="1" date_created="2007-08-29 13:26:20.0" version="" changed_by="1" date_changed="2015-11-05 01:05:41.0" retired_by="1" uuid="1498AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"/>
```

Comment:
You will need to extract form_text ="" into a separate xml

Reply:
ok then